server/events/project_command_runner.go

```diff
@@ -259,18 +276,18 @@ func (p *DefaultProjectCommandRunner) Apply(ctx command.ProjectContext) command.
 	}
 }
 
-func (p *DefaultProjectCommandRunner) ApprovePolicies(ctx command.ProjectContext) command.ProjectResult {
-	approvedOut, failure, err := p.doApprovePolicies(ctx)
-	return command.ProjectResult{
-		Command:            command.PolicyCheck,
-		Failure:            failure,
-		Error:              err,
-		PolicyCheckSuccess: approvedOut,
-		RepoRelDir:         ctx.RepoRelDir,
-		Workspace:          ctx.Workspace,
-		ProjectName:        ctx.ProjectName,
-	}
-}
+//func (p *DefaultProjectCommandRunner) ApprovePolicies(ctx command.ProjectContext) command.ProjectResult {
```

Comment:
Not quite sure what to do here. With the changes in this PR, we need access to the DB state to calculate approval state, which is currently not accessible at this level.

For now I've put all the logic in approve_policies_command_runner.go. Though it feels wrong to break the standard pattern in this case.

Thoughts?